runtime/common/MeasureCounts.cpp

```diff
@@ -484,4 +484,34 @@ bool sample_result::has_even_parity(std::string_view bitString) {
   int c = std::count(bitString.begin(), bitString.end(), '1');
   return c % 2 == 0;
 }
+
+void sample_result::reorder(const std::vector<std::size_t> &idx,
+                            const std::string_view registerName) {
```

Comment:
string_view is non-owning ... that isn't going to cause our friendly StringRef bugs, I hope?

Reply:
I don't think it will because it doesn't attempt save `registerName` anywhere in the data structure.